### PR TITLE
Round Fixes

### PR DESCRIPTION
**Pull request by Ian-Boraks**
Fixing items in issue: https://github.com/EMERALD0874/Steam-Deck-Themes/issues/3

Other fixes as well mentioned in file change comments

### CODE REVIEW

File: Round/shared.css [hunk 1]
```diff
@@ -210,3 +211,35 @@ button.gamepaddialog_Button_1kn70.DialogButton:enabled:active:hover.Primary {
...
+/* Main-Menu Achievements */
+.mainmenuapprunning_OverlayAchievements_2Qced .achievementslist_AchievementListItemBase_2Kmn7,
+.mainmenuapprunning_OverlayAchievements_2Qced .achievementslist_GlobalAchievementListItem_316Uo,
+.mainmenuapprunning_OverlayAchievements_2Qced .achievementslist_AchievementList_1UXC2,
+.mainmenuapprunning_OverlayAchievements_2Qced .achievementsheader_StatsBlock__gDkI  {
+  border-radius: 20px;
+}
+
+/* Exit Game Dialog */
+.gamepaddialog_GamepadDialogContent_3joNk {
```
Comment (Ian-Boraks) [inline Round/shared.css:229]: This is the main item listed in the issue

File: Round/shared.css [hunk 2]
```diff
@@ -210,3 +211,35 @@ button.gamepaddialog_Button_1kn70.DialogButton:enabled:active:hover.Primary {
...
+}
+
+/* Settings */
+.DialogCheckbox_Container,
+.gamepadslider_EditableValue_2DXnl  {
+  border-radius: 20px !important;
+}
+
+.controllerconfigurator_interstitial_ControlLabel_tGJeF {
+  border-radius: 20px;
```
Comment (Ian-Boraks) [inline Round/shared.css:240]: This is the buttons inside the controller configuration tutorial

File: Round/shared.css [hunk 3]
```diff
@@ -210,3 +211,35 @@ button.gamepaddialog_Button_1kn70.DialogButton:enabled:active:hover.Primary {
...
+
+/* Exit Game Dialog */
+.gamepaddialog_GamepadDialogContent_3joNk {
+  border-radius: 20px;
+}
+
+/* Settings */
+.DialogCheckbox_Container,
+.gamepadslider_EditableValue_2DXnl  {
+  border-radius: 20px !important;
```
Comment (Ian-Boraks) [inline Round/shared.css:236]: This is the check-boxes container as well as the container for the value of sliders

File: Round/shared.css [hunk 4]
```diff
@@ -210,3 +211,35 @@ button.gamepaddialog_Button_1kn70.DialogButton:enabled:active:hover.Primary {
...
+}
+
+.controllerconfigurator_interstitial_ControlLabel_tGJeF {
+  border-radius: 20px;
+}
+
+.appproperties_AppProperties_1FyBL .DialogBody .appproperties_TopGapSmall_2evqi {
+  margin-top: 0px !important;
```
Comment (Ian-Boraks) [inline Round/shared.css:244]: This is to address clipping with the text boxes in the setting menus

File: Round/shared.css [hunk 5]
```diff
@@ -210,3 +211,35 @@ button.gamepaddialog_Button_1kn70.DialogButton:enabled:active:hover.Primary {
   border-radius: 20px !important;
   overflow: hidden;
 }
+
+/* Main-Menu Guides */
+.overlayguides_GuideListItemBase_1etIu {
+  border-radius: 20px;
```
Comment (Ian-Boraks) [inline Round/shared.css:217]: This is for the guide containers in the new beta

File: Round/shared.css [hunk 6]
```diff
@@ -210,3 +211,35 @@ button.gamepaddialog_Button_1kn70.DialogButton:enabled:active:hover.Primary {
...
+}
+
+/* Main-Menu Achievements */
+.mainmenuapprunning_OverlayAchievements_2Qced .achievementslist_AchievementListItemBase_2Kmn7,
+.mainmenuapprunning_OverlayAchievements_2Qced .achievementslist_GlobalAchievementListItem_316Uo,
+.mainmenuapprunning_OverlayAchievements_2Qced .achievementslist_AchievementList_1UXC2,
+.mainmenuapprunning_OverlayAchievements_2Qced .achievementsheader_StatsBlock__gDkI  {
+  border-radius: 20px;
```
Comment (Ian-Boraks) [inline Round/shared.css:225]: These are for the achievment containers in the main-menu achiement area

**Review by EMERALD0874 — APPROVED** [review]
LGTM

**Pull request merged**